packages/gcr_3.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gcr_3 < Package
+class Gcr_3 < Meson
   description 'GNOME crypto package'
   homepage 'https://www.gnome.org'
   version '3.41.1'
```

Comment:
Update to [4.2.1](https://gitlab.gnome.org/GNOME/gcr/-/tags/4.2.1)?

Reply:
That's in gcr_4

Reply:
Corrected for gcr_3.